src/components/BoardControls/BoardControls.tsx

```diff
@@ -31,17 +34,20 @@ export class BoardControls extends React.Component<BoardControlsProps, BoardCont
     this.titleInput = React.createRef();
   }
 
-  componentWillMount() {
-    this.props.socket.on(`board:loaded:${this.props.boardId}`, (data: any) => {
-      this.setState({
-        description: data.description,
-        title: data.title,
-      });
+  componentDidMount() {
```

Comment:
Updated to `componentDidMount`, here and in a few of other places as it seems that `componentWillMount` will be removed in React 17 and the most recent docs indicate that subscriptions should be made in `componentDidMount` now. I don't understand "why" at this time, I'm mostly trusting [the docs](https://reactjs.org/docs/react-component.html#unsafe_componentwillmount).

Re: componentWillMount()
>Avoid introducing any side-effects or subscriptions in this method. For those use cases, use `componentDidMount()` instead.